pkg/apis/apps/v1alpha1/apimanager_types.go

```diff
@@ -220,22 +221,30 @@ type SystemFileStorageSpec struct {
 	// +optional
 	PVC *SystemPVCSpec `json:"persistentVolumeClaim,omitempty"`
 	// +optional
-	S3 *SystemS3Spec `json:"amazonSimpleStorageService,omitempty"`
+	// Deprecated
+	DeprecatedS3 *DeprecatedSystemS3Spec `json:"amazonSimpleStorageService,omitempty"`
+	// +optional
+	S3 *SystemS3Spec `json:"simpleStorageService,omitempty"`
```

Comment:
The changes here have been:

We maintain amazonSimpleStorageService json field but we rename the old S3 *SystemS3Spec field and struct type to DeprecatedS3 *DeprecatedSystemS3Spec, and we create a new field S3 and struct type SystemS3Spec with the new attribute, but using a new json name called "simpleStorageService"